mdaviz/mda_file_table_view.py

```diff
@@ -128,16 +128,16 @@ def data2Plot(self, selections):
         Extracts selected datasets for plotting from scanDict based on user selections.
 
         Parameters:
-        - selections: A dictionary with keys "X" and "Y", where "X" is the index for the x-axis data
-          and "Y" is a list of indices for the y-axis data.
+            - selections: A dictionary with keys "X" and "Y", where "X" is the index for the x-axis data
+              and "Y" is a list of indices for the y-axis data.
 
         Returns:
-        - A tuple of (datasets, plot_options), where datasets is a list of tuples containing the
-          data and options (label) for each dataset, and plot_options contains overall plotting configurations.
+            - A tuple of (datasets, plot_options), where datasets is a list of tuples containing the
+              data and options (label) for each dataset, and plot_options contains overall plotting configurations.
 
-        ..note::
-        scanDict = {index: {'object': scanObject, 'data': [...], 'unit': '...', 'name': '...',
-            'type':...}}.
+        .. note::
+
+        scanDict = {index: {'object': scanObject, 'data': [...], 'unit': '...', 'name': '...','type':...}}.
```

Comment:
Should this be indented, too?  It's a code example for the note.

Also, the `.. note::` [directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions) probably requires some text to follow on the same line.

Reply:
Yup.  There's an error about it in the [logs](https://productionresultssa4.blob.core.windows.net/actions-results/000ba645-9c1b-4ddb-92c2-04f686815127/workflow-job-run-7e620c85-24a8-5ffa-8b1f-642bc9b1fc36/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-05-29T20%3A22%3A41Z&sig=q5frnBOybtrEPyKrx2FjXI00DR3UWIzi0cf3tyBwbt8%3D&sp=r&spr=https&sr=b&st=2024-05-29T20%3A12%3A36Z&sv=2021-12-02):

```
2024-05-29T20:09:10.2820840Z /home/runner/work/mdaviz/mdaviz/mdaviz/mda_file_table_view.py:docstring of mdaviz.mda_file_table_view.MDAFileTableView.data2Plot:11: ERROR: Content block expected for the "note" directive; none found.
```

Reply:
If you just want to show computer code, it's easier:

```
::

    E = m * c^2
```

Reply:
I just removed the note. I don't think it is necessary.